.travis.yml

```diff
@@ -3,13 +3,15 @@ language: php
 sudo: false
 
 php:
+  - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
 matrix:
   include:
-    - php: 5.6
+    - php: 5.5
```

Comment:
perhaps we should make this a global env check:

```yaml
env:
  - 'COMPOSER_FLAGS="-- prefer-stable --prefer-lowest"'
  - 'COMPOSER_FLAGS=""'
```

rather than just for 5.5/5.6...